src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -1003,7 +1003,7 @@ See [Oracle's field detail documentation](https://docs.oracle.com/javase/8/docs/
 
 You can use the correlation assistant to more quickly analyze [incidents](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence#issue-feed), create decision logic, and test the logic with a simulation. To use the correlation assistant:
 
-1. From **one.newrelic.com**, click **Alerts & AI**, click **Issues & Activity**, then click the **Incidents** tab.
+1. From **[one.newrelic.com](https://one.newrelic.com)**, click **Alerts & AI**, click **Issues & Activity**, then click the **Incidents** tab.
```

Comment:
This might be one we could switch to the greater-than navigation: Go to **[one.newrelic.com](https://one.newrelic.com) > Alerts & AI > Issues & Activity > Incidents**.